project/src/Agents/GSTNodeAgent.gd

```diff
@@ -0,0 +1,79 @@
+# A base class for a specialized steering agent that updates itself every frame
```

Comment:
The docstring should clearly distinguish this class from the Node2D one. This is the base, abstract type for self-updating agents, correct? Maybe we can add an Abstract tag in the docs system

Reply:
Yeah. I'll clarify the docstring.

Though an abstract tag wouldn't go unappreciated.

Reply:
You can already add the tag: abstract, I'll patch docs maker in a second

Reply:
I added it in the refactor ♥